packages/atlas/src/components/_inputs/RadioButton/RadioButton.stories.tsx

```diff
@@ -1,32 +1,27 @@
 import { Meta, Story } from '@storybook/react'
-import React, { useState } from 'react'
+import React from 'react'
 
-import { RadioButton } from './RadioButton'
+import { RadioButton, RadioButtonProps } from './RadioButton'
 
 export default {
   title: 'inputs/RadioButton',
   component: RadioButton,
+  argTypes: {
+    className: { table: { disable: true } },
+    selectedValue: { table: { disable: true } },
+    selected: { controls: { type: 'boolean' } },
+  },
   args: {
     label: 'Hello there',
     helperText: 't100',
```

Comment:
![image](https://user-images.githubusercontent.com/51168865/173002141-ed276d1c-b741-46b0-a988-ea7ab29c250c.png)
We still have `helperText` control which does nothing. I'm a little surprised why typescript doesn't mark that as an error.